versioneer.py

```diff
@@ -1404,6 +1405,12 @@ def get_versions(verbose=False):
 
     Returns dict with two keys: 'version' and 'full'.
     """
+    ver_file = Path(__file__).parent / 'fmriprep' / 'VERSION'
+
+    if ver_file.is_file():
+        ver = ver_file.read_text().splitlines()[0].strip()
+        return {"version": ver, 'full': ver}
+
```

Comment:
I think this part probably isn't necessary. Since we don't distribute containers, we don't really need the `VERSION` hack.

Reply:
probably not, I agree